src/colvarproxy_tcl.cpp

```diff
@@ -36,8 +35,18 @@ colvarproxy_tcl::~colvarproxy_tcl()
 
 void colvarproxy_tcl::init_tcl_pointers()
 {
-  cvm::error("Error: Tcl support is not available in this build.\n",
-             COLVARS_NOT_IMPLEMENTED);
+  // This is overloaded by NAMD and VMD proxies to use the local interpreters
```

Comment:
Currently this is only overloaded by NAMD

Reply:
It's also overloaded in VMD, by a function that does nothing.